foundry.toml

```diff
@@ -0,0 +1,24 @@
+[profile.default]
+src = "src"
+out = "out"
+libs = ["node_modules", "lib"]
+remappings = [
+    "@ensdomains/=node_modules/@ensdomains/",
+    "@gnosis.pm/=node_modules/@gnosis.pm/",
+    "@openzeppelin/=node_modules/@openzeppelin/",
+    "@uniswap/=node_modules/@uniswap/",
+    "canonical-weth/=node_modules/canonical-weth/",
+    "eth-gas-reporter/=node_modules/eth-gas-reporter/",
+    "hardhat-deploy/=node_modules/hardhat-deploy/",
+    "hardhat/=node_modules/hardhat/",
+]
+
+# Compiler settings
+solc = "0.7.6"
+bytecode_hash = "none"
```

Comment:
We might have an issue:
https://book.getfoundry.sh/forge/deploying?highlight=bytecode_hash#verifying-contracts-with-no-bytecode-hash

I'd leave this out until this is sorted out and/or deployment scripts are written. In any case, if included, I'd add a comment with its purpose.

```suggestion
```